server/src/postgres/init.rs

```diff
@@ -17,6 +18,9 @@ use crate::resources::{
 /// This function makes sure the tables will be created in the correct order
 pub async fn create_db() -> Result<(), ArrErr> {
     psql_info!("Creating database tables.");
```

Comment:
Some log statements could use the `(fn prepend)` format, see below:

<details>

Test | /// | Location                                                             | Type                 | Statement                                         
---- | --- | -------------------------------------------------------------------- | -------------------- | --------------------------------------------------
 X   |  X  | server/src/main.rs (19)                                              | (no_fn_prepend) msg  | info!("Running database initialization");         
 X   |  X  | server/src/main.rs (19)                                              | punctuation          | info!("Running database initialization");         
 X   |  X  | server/src/main.rs (27)                                              | (no_fn_prepend) msg  | info!("Found argument [init_psql]. Creating dat...
 X   |  X  | server/src/main.rs (29)                                              | (no_fn_prepend) msg  | info!("PSQL Database creation completed.");       
 X   |  X  | server/src/main.rs (34)                                              | (no_fn_prepend) msg  | info!("Found argument [rebuild_psql]. Rebuildin...
 X   |  X  | server/src/main.rs (36)                                              | (no_fn_prepend) msg  | info!("PSQL Rebuild completed.");                 
 X   |  X  | server/src/main.rs (42)                                              | (no_fn_prepend) msg  | info!("Server shutdown.");                        
 X   |  X  | server/src/config.rs (32)                                            | (no_fn_prepend) msg  | log::warn!("Creating Config object with default...
 X   |  X  | server/src/lib.rs (90)                                               | punctuation          | log::warn!("(shutdown_signal) server shutdown for 
 X   |  X  | server/src/postgres/linked_resource.rs (25)                          | punctuation          | psql_debug!("(get_for_ids) start: [{:?}]", ids);  
 X   |  X  | server/src/postgres/linked_resource.rs (38)                          | punctuation          | psql_debug!("(delete_for_ids) start: [{:?}]", ids)
 X   |  X  | server/src/postgres/linked_resource.rs (58)                          | (no_fn_prepend) msg  | psql_debug!("{}", &query);                        
 X   |  X  | server/src/postgres/linked_resource.rs (59)                          | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/linked_resource.rs (64)                          | (no_fn_prepend) msg  | psql_info!(            "Deleting rows for table...
 X   |  X  | server/src/postgres/linked_resource.rs (81)                          | (no_fn_prepend) msg  | psql_debug!(                            "Remove...
 X   |  X  | server/src/postgres/linked_resource.rs (81)                          | punctuation          | psql_debug!(                            "Removed [
 X   |  X  | server/src/postgres/linked_resource.rs (96)                          | (no_fn_prepend) msg  | psql_debug!(                            "Remove...
 X   |  X  | server/src/postgres/linked_resource.rs (96)                          | punctuation          | psql_debug!(                            "Removed [
 X   |  X  | server/src/postgres/linked_resource.rs (113)                         | punctuation          | psql_debug!("(link_ids) start: [{:?}] replace [{:?
 X   |  X  | server/src/postgres/linked_resource.rs (145)                         | (no_fn_prepend) msg  | psql_debug!("{}", insert_sql);                    
 X   |  X  | server/src/postgres/linked_resource.rs (146)                         | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/linked_resource.rs (148)                         | (no_fn_prepend) msg  | psql_info!("Update/Insert entry for table [{}]....
 X   |  X  | server/src/postgres/linked_resource.rs (174)                         | punctuation          | psql_debug!("(read) start: [{:?}]", self.try_get_u
 X   |  X  | server/src/postgres/linked_resource.rs (193)                         | (no_fn_prepend) msg  | psql_debug!("{}", query);                         
 X   |  X  | server/src/postgres/linked_resource.rs (194)                         | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/linked_resource.rs (203)                         | (no_fn_prepend) msg  | psql_info!(            "Fetching row data for t...
 X   |  X  | server/src/postgres/linked_resource.rs (216)                         | punctuation          | psql_debug!("(delete) start: [{:?}]", self.try_get
 X   |  X  | server/src/postgres/linked_resource.rs (224)                         | (no_fn_prepend) msg  | psql_info!(            "Deleting entry from tab...
 X   |  X  | server/src/postgres/linked_resource.rs (240)                         | (no_fn_prepend) msg  | psql_debug!("{}", query);                         
 X   |  X  | server/src/postgres/linked_resource.rs (241)                         | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/linked_resource.rs (246)                         | (no_fn_prepend) msg  | psql_info!("Removing entry from table [{}].", d...
 X   |  X  | server/src/postgres/linked_resource.rs (264)                         | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/simple_resource_linked.rs (30)                   | punctuation          | psql_debug!("(get_for_ids) start: [{:?}]", ids);  
 X   |  X  | server/src/postgres/simple_resource_linked.rs (43)                   | punctuation          | psql_debug!("(delete_for_ids) start: [{:?}]", ids)
 X   |  X  | server/src/postgres/simple_resource_linked.rs (63)                   | (no_fn_prepend) msg  | psql_debug!("{}", &query);                        
 X   |  X  | server/src/postgres/simple_resource_linked.rs (64)                   | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/simple_resource_linked.rs (69)                   | (no_fn_prepend) msg  | psql_info!(            "Deleting rows for table...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (86)                   | (no_fn_prepend) msg  | psql_debug!(                            "Remove...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (86)                   | punctuation          | psql_debug!(                            "Removed [
 X   |  X  | server/src/postgres/simple_resource_linked.rs (101)                  | (no_fn_prepend) msg  | psql_debug!(                            "Remove...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (101)                  | punctuation          | psql_debug!(                            "Removed [
 X   |  X  | server/src/postgres/simple_resource_linked.rs (118)                  | punctuation          | psql_debug!("(create) start: [{:?}]", row_data);  
 X   |  X  | server/src/postgres/simple_resource_linked.rs (135)                  | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/simple_resource_linked.rs (165)                  | punctuation          | psql_debug!("(read) start: [{:?}]", self.try_get_u
 X   |  X  | server/src/postgres/simple_resource_linked.rs (183)                  | punctuation          | psql_debug!("(update) start: [{:?}]", data);      
 X   |  X  | server/src/postgres/simple_resource_linked.rs (207)                  | (no_fn_prepend) msg  | psql_debug!("{}", update_sql);                    
 X   |  X  | server/src/postgres/simple_resource_linked.rs (209)                  | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/simple_resource_linked.rs (215)                  | (no_fn_prepend) msg  | psql_info!(            "Updating entry in table...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (263)                  | punctuation          | psql_debug!("(set_deleted_at_now) start: [{:?}]", 
 X   |  X  | server/src/postgres/simple_resource_linked.rs (273)                  | (no_fn_prepend) msg  | psql_info!(                "[deleted_at] column...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (283)                  | (no_fn_prepend) msg  | psql_info!(            "Updating [deleted_at] f...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (301)                  | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/simple_resource_linked.rs (319)                  | punctuation          | psql_debug!("(set_deleted_at_now) start: [{:?}]", 
 X   |  X  | server/src/postgres/simple_resource_linked.rs (328)                  | (no_fn_prepend) msg  | psql_info!(            "Deleting entry from tab...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (345)                  | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/simple_resource_linked.rs (368)                  | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/simple_resource_linked.rs (393)                  | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/simple_resource_linked.rs (411)                  | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/simple_resource_linked.rs (429)                  | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/simple_resource_linked.rs (449)                  | (no_fn_prepend) msg  | psql_debug!(                        "Skipping u...
 X   |  X  | server/src/postgres/simple_resource_linked.rs (449)                  | punctuation          | psql_debug!(                        "Skipping upda
 X   |  X  | server/src/postgres/queries.rs (17)                                  | punctuation          | psql_debug!("(get_by_id) start: [{:?}]", id);     
 X   |  X  | server/src/postgres/queries.rs (33)                                  | (no_fn_prepend) msg  | psql_debug!("{}", &query);                        
 X   |  X  | server/src/postgres/queries.rs (46)                                  | punctuation          | psql_debug!("(get_for_ids) start: [{:?}]", ids);  
 X   |  X  | server/src/postgres/queries.rs (74)                                  | (no_fn_prepend) msg  | psql_debug!("{}", &query);                        
 X   |  X  | server/src/postgres/queries.rs (75)                                  | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/init.rs (20)                                     | (no_fn_prepend) msg  | psql_info!("Creating database tables.");          
 X   |  X  | server/src/postgres/init.rs (42)                                     | (no_fn_prepend) msg  | psql_warn!("Dropping database tables.");          
 X   |  X  | server/src/postgres/init.rs (64)                                     | (no_fn_prepend) msg  | psql_warn!("Re-creating database tables.");       
 X   |  X  | server/src/postgres/init.rs (88)                                     | (no_fn_prepend) msg  | psql_debug!("{}", index_query);                   
 X   |  X  | server/src/postgres/init.rs (94)                                     | (no_fn_prepend) msg  | psql_error!(                    "Failed to crea...
 X   |  X  | server/src/postgres/init.rs (107)                                    | (no_fn_prepend) msg  | psql_debug!("{}", create_table);                  
 X   |  X  | server/src/postgres/init.rs (109)                                    | (no_fn_prepend) msg  | psql_error!("Failed to create table: {}", e);     
 X   |  X  | server/src/postgres/init.rs (123)                                    | (no_fn_prepend) msg  | psql_debug!("{}", drop_query);                    
 X   |  X  | server/src/postgres/init.rs (125)                                    | (no_fn_prepend) msg  | psql_info!("Dropping table [{}].", definition.p...
 X   |  X  | server/src/postgres/init.rs (127)                                    | (no_fn_prepend) msg  | psql_error!("Failed to drop table [{}]: {}", e,...
 X   |  X  | server/src/postgres/init.rs (151)                                    | (no_fn_prepend) msg  | psql_info!(            "composing create table ...
 X   |  X  | server/src/postgres/init.rs (151)                                    | punctuation          | psql_info!(            "composing create table que
 X   |  X  | server/src/postgres/init.rs (189)                                    | (no_fn_prepend) msg  | psql_info!(            "composing create table ...
 X   |  X  | server/src/postgres/init.rs (189)                                    | punctuation          | psql_info!(            "composing create table que
 X   |  X  | server/src/postgres/search.rs (107)                                  | (no_fn_prepend) msg  | psql_error!(                        "Invalid fi...
 X   |  X  | server/src/postgres/search.rs (107)                                  | punctuation          | psql_error!(                        "Invalid field
 X   |  X  | server/src/postgres/search.rs (126)                                  | (no_fn_prepend) msg  | psql_info!(            "Searching table [{}] wi...
 X   |  X  | server/src/postgres/search.rs (126)                                  | punctuation          | psql_info!(            "Searching table [{}] with 
 X   |  X  | server/src/postgres/search.rs (127)                                  | (no_fn_prepend) msg  | psql_debug!("Params: {:?}", params);              
 X   |  X  | server/src/postgres/search.rs (149)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (161)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (172)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (191)                                  | (no_fn_prepend) msg  | psql_debug!("Found [{}] filter", operator.as_st...
 X   |  X  | server/src/postgres/search.rs (191)                                  | punctuation          | psql_debug!("Found [{}] filter", operator.as_str_n
 X   |  X  | server/src/postgres/search.rs (387)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (399)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (410)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (421)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (432)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (443)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (454)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/search.rs (465)                                  | (no_fn_prepend) msg  | psql_error!("{}", err);                           
 X   |  X  | server/src/postgres/mod.rs (58)                                      | (no_fn_prepend) msg  | psql_error!("error executing DB query: {}", err); 
 X   |  X  | server/src/postgres/mod.rs (65)                                      | (no_fn_prepend) msg  | psql_error!("postgres pool error: {}", err);      
 X   |  X  | server/src/postgres/mod.rs (72)                                      | (no_fn_prepend) msg  | psql_error!("postgres pool config error: {}", e...
 X   |  X  | server/src/postgres/mod.rs (117)                                     | (no_fn_prepend) msg  | psql_debug!(            "Creating PostgresPool ...
 X   |  X  | server/src/postgres/mod.rs (120)                                     | (no_fn_prepend) msg  | psql_info!("Initializing connection with TLS se...
 X   |  X  | server/src/postgres/mod.rs (120)                                     | punctuation          | psql_info!("Initializing connection with TLS setti
 X   |  X  | server/src/postgres/mod.rs (121)                                     | (no_fn_prepend) msg  | psql_debug!("{:?}", settings);                    
 X   |  X  | server/src/postgres/mod.rs (122)                                     | (no_fn_prepend) msg  | psql_info!("Try read root cert file: {}", setti...
 X   |  X  | server/src/postgres/mod.rs (130)                                     | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/mod.rs (134)                                     | (no_fn_prepend) msg  | psql_info!("Try load root cert file.");           
 X   |  X  | server/src/postgres/mod.rs (142)                                     | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/mod.rs (146)                                     | (no_fn_prepend) msg  | psql_debug!("Root cert load success.");           
 X   |  X  | server/src/postgres/mod.rs (159)                                     | (no_fn_prepend) msg  | psql_info!("Try read client cert file.");         
 X   |  X  | server/src/postgres/mod.rs (165)                                     | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/mod.rs (168)                                     | (no_fn_prepend) msg  | psql_info!("Try read client key file.");          
 X   |  X  | server/src/postgres/mod.rs (174)                                     | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/mod.rs (178)                                     | (no_fn_prepend) msg  | psql_info!("Setting up TLS connection with clie...
 X   |  X  | server/src/postgres/mod.rs (196)                                     | (no_fn_prepend) msg  | psql_info!("Setting up TLS connection with clie...
 X   |  X  | server/src/postgres/mod.rs (211)                                     | (no_fn_prepend) msg  | psql_warn!("Setting up database connection with...
 X   |  X  | server/src/postgres/mod.rs (215)                                     | (no_fn_prepend) msg  | psql_info!("Successfully created PostgresPool."); 
 X   |  X  | server/src/postgres/mod.rs (221)                                     | (no_fn_prepend) msg  | psql_debug!("Checking database readiness.");      
 X   |  X  | server/src/postgres/mod.rs (234)                                     | (no_fn_prepend) msg  | psql_debug!("Success, the database is ready.");   
 X   |  X  | server/src/postgres/mod.rs (245)                                     | (no_fn_prepend) msg  | psql_info!("Initializing global shared psql poo...
 X   |  X  | server/src/postgres/mod.rs (248)                                     | (no_fn_prepend) msg  | println!("Initializing database connection pool...
 X   |  X  | server/src/postgres/simple_resource.rs (24)                          | punctuation          | psql_debug!(            "(try_get_id_field) start:
 X   |  X  | server/src/postgres/simple_resource.rs (31)                          | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/simple_resource.rs (39)                          | punctuation          | psql_debug!("(get_by_id) start: [{:?}]", id);     
 X   |  X  | server/src/postgres/simple_resource.rs (51)                          | punctuation          | psql_debug!("(create) start: [{:?}]", data);      
 X   |  X  | server/src/postgres/simple_resource.rs (97)                          | punctuation          | psql_debug!("(read) start: [{:?}]", self.try_get_u
 X   |  X  | server/src/postgres/simple_resource.rs (115)                         | punctuation          | psql_debug!("(update) start: [{:?}]", data);      
 X   |  X  | server/src/postgres/simple_resource.rs (139)                         | (no_fn_prepend) msg  | psql_debug!("{}", update_sql);                    
 X   |  X  | server/src/postgres/simple_resource.rs (141)                         | (no_fn_prepend) msg  | psql_debug!("{:?}", &params);                     
 X   |  X  | server/src/postgres/simple_resource.rs (147)                         | (no_fn_prepend) msg  | psql_info!(            "Updating entry in table...
 X   |  X  | server/src/postgres/simple_resource.rs (195)                         | punctuation          | psql_debug!("(set_deleted_at_now) start: [{:?}]", 
 X   |  X  | server/src/postgres/simple_resource.rs (205)                         | (no_fn_prepend) msg  | psql_info!(                "[deleted_at] column...
 X   |  X  | server/src/postgres/simple_resource.rs (215)                         | (no_fn_prepend) msg  | psql_info!(            "Updating [deleted_at] f...
 X   |  X  | server/src/postgres/simple_resource.rs (233)                         | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/simple_resource.rs (251)                         | punctuation          | psql_debug!("(set_deleted_at_now) start: [{:?}]", 
 X   |  X  | server/src/postgres/simple_resource.rs (260)                         | (no_fn_prepend) msg  | psql_info!(            "Deleting entry from tab...
 X   |  X  | server/src/postgres/simple_resource.rs (277)                         | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (29)                                     | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (54)                                     | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (75)                                     | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (92)                                     | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (104)                                    | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (124)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (144)                                    | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (159)                                    | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (202)                                    | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (214)                                    | (no_fn_prepend) msg  | psql_info!("{}", error);                          
 X   |  X  | server/src/postgres/util.rs (241)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (268)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (286)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (304)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (324)                                    | (no_fn_prepend) msg  | psql_debug!(                    "Skipping inser...
 X   |  X  | server/src/postgres/util.rs (324)                                    | punctuation          | psql_debug!(                    "Skipping insert [
 X   |  X  | server/src/postgres/util.rs (350)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (375)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (393)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (411)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (431)                                    | (no_fn_prepend) msg  | psql_debug!(                    "Skipping updat...
 X   |  X  | server/src/postgres/util.rs (431)                                    | punctuation          | psql_debug!(                    "Skipping update [
 X   |  X  | server/src/postgres/util.rs (526)                                    | punctuation          | psql_debug!("(validate) start: [{:?}]", data);    
 X   |  X  | server/src/postgres/util.rs (550)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (560)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (646)                                    | (no_fn_prepend) msg  | psql_error!("{}", error);                         
 X   |  X  | server/src/postgres/util.rs (654)                                    | (no_fn_prepend) msg  | psql_debug!("fields provided: {:?}", data);       
 X   |  X  | server/src/postgres/util.rs (655)                                    | (no_fn_prepend) msg  | psql_debug!("errors found: {:?}", errors);        
 X   |  X  | server/src/postgres/util.rs (660)                                    | (no_fn_prepend) msg  | psql_info!("{}", info);                           
 X   |  X  | server/src/resources/vertiport/mod.rs (99)                           | (no_fn_prepend) msg  | debug!("Converting Row to vertiport::Data: {:?}...
 X   |  X  | server/src/resources/adsb/mod.rs (72)                                | (no_fn_prepend) msg  | debug!("Converting Row to adsb::Data: {:?}", row);
 X   |  X  | server/src/resources/itinerary/mod.rs (78)                           | (no_fn_prepend) msg  | debug!("Converting Row to itinerary::Data: {:?}...
 X   |  X  | server/src/resources/itinerary/flight_plan.rs (37)                   | (no_fn_prepend) msg  | debug!("Converting Row to itinerary_flight_plan...
 X   |  X  | server/src/resources/vehicle/mod.rs (135)                            | (no_fn_prepend) msg  | debug!("Converting Row to vehicle::Data: {:?}",...
 X   |  X  | server/src/resources/scanner/mod.rs (100)                            | (no_fn_prepend) msg  | debug!("Converting Row to scanner::Data: {:?}",...
 X   |  X  | server/src/resources/group/mod.rs (81)                               | (no_fn_prepend) msg  | debug!("Converting Row to group::Data: {:?}", r...
 X   |  X  | server/src/resources/parcel/mod.rs (92)                              | (no_fn_prepend) msg  | debug!("Converting Row to parcel::Data: {:?}", ...
 X   |  X  | server/src/resources/base/simple_resource_linked.rs (81)             | (no_fn_prepend) msg  | debug!("Converting Vec<Row> to List: {:?}", rows);
 X   |  X  | server/src/resources/base/simple_resource_linked.rs (107)            | (no_fn_prepend) msg  | debug!("Converting Vec<Row> to RowDataList: {:?...
 X   |  X  | server/src/resources/base/simple_resource_linked.rs (146)            | (no_fn_prepend) msg  | debug!("No ids found when converting ResourceOb...
 X   |  X  | server/src/resources/base/simple_resource_linked.rs (146)            | punctuation          | debug!("No ids found when converting ResourceObjec
 X   |  X  | server/src/resources/base/mod.rs (80)                                | (no_fn_prepend) msg  | error!("{}", error);                              
 X   |  X  | server/src/resources/base/mod.rs (320)                               | (no_fn_prepend) msg  | error!("{}", error);                              
 X   |  X  | server/src/resources/base/simple_resource.rs (55)                    | (no_fn_prepend) msg  | error!(                    "Could not set id fo...
 X   |  X  | server/src/resources/base/simple_resource.rs (55)                    | punctuation          | error!(                    "Could not set id for R
 X   |  X  | server/src/resources/base/simple_resource.rs (70)                    | (no_fn_prepend) msg  | error!("{}", error);                              
 X   |  X  | server/src/resources/base/simple_resource.rs (161)                   | (no_fn_prepend) msg  | debug!("Converting Vec<Row> to List: {:?}", rows);
 X   |  X  | server/src/resources/user/mod.rs (88)                                | (no_fn_prepend) msg  | debug!("Converting Row to user::Data: {:?}", row);
 X   |  X  | server/src/resources/user/group.rs (37)                              | (no_fn_prepend) msg  | debug!("Converting Row to user_group::Data: {:?...
 X   |  X  | server/src/resources/parcel_scan/mod.rs (92)                         | (no_fn_prepend) msg  | debug!("Converting Row to parcel::Data: {:?}", ...
 X   |  X  | server/src/resources/flight_plan/parcel.rs (60)                      | (no_fn_prepend) msg  | debug!("Converting Row to flight_plan_parcel::D...
 X   |  X  | server/src/resources/flight_plan/parcel.rs (91)                      | (no_fn_prepend) msg  | debug!("Converting Row to flight_plan_parcel::D...
 X   |  X  | server/src/resources/flight_plan/mod.rs (200)                        | (no_fn_prepend) msg  | debug!("Converting Row to flight_plan::Data: {:...
 X   |  X  | server/src/resources/pilot/mod.rs (74)                               | (no_fn_prepend) msg  | debug!("Converting Row to pilot::Data: {:?}", r...
 X   |  X  | server/src/resources/vertipad/mod.rs (112)                           | (no_fn_prepend) msg  | debug!("Converting Row to vertipad::Data: {:?}"...
 X   |  X  | server/src/grpc/link_service.rs (124)                                | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/link_service.rs (134)                                | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/link_service.rs (174)                                | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/link_service.rs (245)                                | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/mod.rs (120)                                         | (no_fn_prepend) msg  | grpc_warn!("{:#}", err);                          
 X   |  X  | server/src/grpc/simple_service.rs (76)                               | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service.rs (120)                              | (no_fn_prepend) msg  | grpc_debug!("Inserting with data {:?}", resourc...
 X   |  X  | server/src/grpc/simple_service.rs (134)                              | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service.rs (135)                              | (no_fn_prepend) msg  | grpc_debug!("{:?}", resource.try_get_data()?);    
 X   |  X  | server/src/grpc/simple_service.rs (136)                              | (no_fn_prepend) msg  | grpc_debug!("{:?}", validation_result);           
 X   |  X  | server/src/grpc/simple_service.rs (174)                              | (no_fn_prepend) msg  | grpc_error!("{}", err);                           
 X   |  X  | server/src/grpc/simple_service.rs (190)                              | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service.rs (191)                              | (no_fn_prepend) msg  | grpc_debug!("{:?}", data);                        
 X   |  X  | server/src/grpc/simple_service.rs (192)                              | (no_fn_prepend) msg  | grpc_debug!("{:?}", validation_result);           
 X   |  X  | server/src/grpc/server.rs (154)                                      | (no_fn_prepend) msg  | grpc_error!("Failed to parse gRPC address: {}",...
 X   |  X  | server/src/grpc/simple_service_linked.rs (145)                       | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service_linked.rs (191)                       | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service_linked.rs (270)                       | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service_linked.rs (319)                       | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service_linked.rs (320)                       | (no_fn_prepend) msg  | grpc_debug!("{:?}", data);                        
 X   |  X  | server/src/grpc/simple_service_linked.rs (321)                       | (no_fn_prepend) msg  | grpc_debug!("{:?}", validation_result);           
 X   |  X  | server/src/grpc/simple_service_linked.rs (358)                       | (no_fn_prepend) msg  | grpc_error!("{}", err);                           
 X   |  X  | server/src/grpc/simple_service_linked.rs (374)                       | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | server/src/grpc/simple_service_linked.rs (375)                       | (no_fn_prepend) msg  | grpc_debug!("{:?}", data);                        
 X   |  X  | server/src/grpc/simple_service_linked.rs (376)                       | (no_fn_prepend) msg  | grpc_debug!("{:?}", validation_result);           
 X   |  X  | server/src/grpc/macros.rs (173)                                      | (no_fn_prepend) msg  | grpc_error!("{}", error);                         
 X   |  X  | client-grpc/includes/build.rs (81)                                   | (no_fn_prepend) msg  | println!("cargo:rustc-env=OUT_DIR={}", out_path); 
 X   |  X  | client-grpc/includes/search.rs (756)                                 | (no_fn_prepend) msg  | println!(                    "Equals filter wit...
 X   |  X  | client-grpc/includes/search.rs (756)                                 | punctuation          | println!(                    "Equals filter with v
 X   |  X  | client-grpc/includes/search.rs (758)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (768)                                 | (no_fn_prepend) msg  | println!(                    "NotEquals filter ...
 X   |  X  | client-grpc/includes/search.rs (768)                                 | punctuation          | println!(                    "NotEquals filter wit
 X   |  X  | client-grpc/includes/search.rs (770)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (779)                                 | (no_fn_prepend) msg  | println!(                    "In filter with va...
 X   |  X  | client-grpc/includes/search.rs (779)                                 | punctuation          | println!(                    "In filter with value
 X   |  X  | client-grpc/includes/search.rs (782)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (792)                                 | (no_fn_prepend) msg  | println!(                    "NotIn filter with...
 X   |  X  | client-grpc/includes/search.rs (792)                                 | punctuation          | println!(                    "NotIn filter with va
 X   |  X  | client-grpc/includes/search.rs (800)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (808)                                 | (no_fn_prepend) msg  | println!(                    "Between filter wi...
 X   |  X  | client-grpc/includes/search.rs (808)                                 | punctuation          | println!(                    "Between filter with 
 X   |  X  | client-grpc/includes/search.rs (826)                                 | (no_fn_prepend) msg  | println!(                    "Found min [{}] an...
 X   |  X  | client-grpc/includes/search.rs (829)                                 | (no_fn_prepend) msg  | println!("Can convert val to number, got [{}]",...
 X   |  X  | client-grpc/includes/search.rs (829)                                 | punctuation          | println!("Can convert val to number, got [{}]", nu
 X   |  X  | client-grpc/includes/search.rs (837)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (843)                                 | (no_fn_prepend) msg  | println!("Can convert val to date, got [{}]", d...
 X   |  X  | client-grpc/includes/search.rs (843)                                 | punctuation          | println!("Can convert val to date, got [{}]", date
 X   |  X  | client-grpc/includes/search.rs (860)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (867)                                 | (no_fn_prepend) msg  | println!(                        "Can't convert...
 X   |  X  | client-grpc/includes/search.rs (867)                                 | punctuation          | println!(                        "Can't convert va
 X   |  X  | client-grpc/includes/search.rs (871)                                 | (no_fn_prepend) msg  | println!("IsNull filter for value [{}]", val);    
 X   |  X  | client-grpc/includes/search.rs (871)                                 | punctuation          | println!("IsNull filter for value [{}]", val);    
 X   |  X  | client-grpc/includes/search.rs (873)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (878)                                 | (no_fn_prepend) msg  | println!("IsNotNull filter for value [{}]", val); 
 X   |  X  | client-grpc/includes/search.rs (878)                                 | punctuation          | println!("IsNotNull filter for value [{}]", val); 
 X   |  X  | client-grpc/includes/search.rs (880)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (911)                                 | (no_fn_prepend) msg  | println!(                    "Greater filter wi...
 X   |  X  | client-grpc/includes/search.rs (911)                                 | punctuation          | println!(                    "Greater filter with 
 X   |  X  | client-grpc/includes/search.rs (913)                                 | (no_fn_prepend) msg  | println!("Can convert val to number, got [{}]",...
 X   |  X  | client-grpc/includes/search.rs (913)                                 | punctuation          | println!("Can convert val to number, got [{}]", nu
 X   |  X  | client-grpc/includes/search.rs (921)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (927)                                 | (no_fn_prepend) msg  | println!("Can convert val to date, got [{}]", d...
 X   |  X  | client-grpc/includes/search.rs (927)                                 | punctuation          | println!("Can convert val to date, got [{}]", date
 X   |  X  | client-grpc/includes/search.rs (936)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (943)                                 | (no_fn_prepend) msg  | println!(                        "Can't convert...
 X   |  X  | client-grpc/includes/search.rs (943)                                 | punctuation          | println!(                        "Can't convert va
 X   |  X  | client-grpc/includes/search.rs (951)                                 | (no_fn_prepend) msg  | println!(                    "GreaterOrEqual fi...
 X   |  X  | client-grpc/includes/search.rs (951)                                 | punctuation          | println!(                    "GreaterOrEqual filte
 X   |  X  | client-grpc/includes/search.rs (953)                                 | (no_fn_prepend) msg  | println!("Can convert val to number, got [{}]",...
 X   |  X  | client-grpc/includes/search.rs (953)                                 | punctuation          | println!("Can convert val to number, got [{}]", nu
 X   |  X  | client-grpc/includes/search.rs (961)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (967)                                 | (no_fn_prepend) msg  | println!("Can convert val to date, got [{}]", d...
 X   |  X  | client-grpc/includes/search.rs (967)                                 | punctuation          | println!("Can convert val to date, got [{}]", date
 X   |  X  | client-grpc/includes/search.rs (976)                                 | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (983)                                 | (no_fn_prepend) msg  | println!(                        "Can't convert...
 X   |  X  | client-grpc/includes/search.rs (983)                                 | punctuation          | println!(                        "Can't convert va
 X   |  X  | client-grpc/includes/search.rs (991)                                 | (no_fn_prepend) msg  | println!(                    "Less filter with ...
 X   |  X  | client-grpc/includes/search.rs (991)                                 | punctuation          | println!(                    "Less filter with val
 X   |  X  | client-grpc/includes/search.rs (993)                                 | (no_fn_prepend) msg  | println!("Can convert val to number, got [{}]",...
 X   |  X  | client-grpc/includes/search.rs (993)                                 | punctuation          | println!("Can convert val to number, got [{}]", nu
 X   |  X  | client-grpc/includes/search.rs (1001)                                | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (1007)                                | (no_fn_prepend) msg  | println!("Can convert val to date, got [{}]", d...
 X   |  X  | client-grpc/includes/search.rs (1007)                                | punctuation          | println!("Can convert val to date, got [{}]", date
 X   |  X  | client-grpc/includes/search.rs (1016)                                | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (1023)                                | (no_fn_prepend) msg  | println!(                        "Can't convert...
 X   |  X  | client-grpc/includes/search.rs (1023)                                | punctuation          | println!(                        "Can't convert va
 X   |  X  | client-grpc/includes/search.rs (1031)                                | (no_fn_prepend) msg  | println!(                    "LessOrEqual filte...
 X   |  X  | client-grpc/includes/search.rs (1031)                                | punctuation          | println!(                    "LessOrEqual filter w
 X   |  X  | client-grpc/includes/search.rs (1033)                                | (no_fn_prepend) msg  | println!("Can convert val to number, got [{}]",...
 X   |  X  | client-grpc/includes/search.rs (1033)                                | punctuation          | println!("Can convert val to number, got [{}]", nu
 X   |  X  | client-grpc/includes/search.rs (1041)                                | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (1047)                                | (no_fn_prepend) msg  | println!("Can convert val to date, got [{}]", d...
 X   |  X  | client-grpc/includes/search.rs (1047)                                | punctuation          | println!("Can convert val to date, got [{}]", date
 X   |  X  | client-grpc/includes/search.rs (1056)                                | (no_fn_prepend) msg  | println!("found!");                               
 X   |  X  | client-grpc/includes/search.rs (1063)                                | (no_fn_prepend) msg  | println!(                        "Can't convert...
 X   |  X  | client-grpc/includes/search.rs (1063)                                | punctuation          | println!(                        "Can't convert va
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (112)                       | (no_fn_prepend) msg  | println!(            "departure_date {} is in t...
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (129)                       | (no_fn_prepend) msg  | println!(            "now {} departure_date {} ...
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (129)                       | punctuation          | println!(            "now {} departure_date {} are
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (142)                       | (no_fn_prepend) msg  | println!(            "now {} departure_date {} ...
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (142)                       | punctuation          | println!(            "now {} departure_date {} are
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (149)                       | (no_fn_prepend) msg  | println!(            "departure_date {} is in t...
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (156)                       | (no_fn_prepend) msg  | println!(            "arrival_date {} is in the...
 X   |  X  | client-grpc/includes/flight_plan/mock.rs (161)                       | (no_fn_prepend) msg  | println!(            "arrival_date {} is in the...

 </details>

You can use the script I linked in the DevOps forum, allows you to click on the filename to jump directly